### PR TITLE
added lab4 with Printer class

### PR DESCRIPTION
**Pull request by IrynaLutsan**
My first pull request

### CODE REVIEW

File: src/printer_lab.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    
+    def set_speed(self, speed):
+        self.__speed = speed
+    
+    def get_price(self):
+        return self.__price
+    
+    def set_price(self, price):
+        self.__price = price
+
```
Comment (victoriiaa1) [inline src/printer_lab.py:32]: take 1 ribbon

File: src/printer_lab.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    def set_price(self, price):
+        self.__price = price
+    
+    
+    def __str__(self):
+        return Printer.manufacturer + ": " + self.__name + " / " + str(self.__speed) + " ppm " + "(" + str(self.__price) + " грн" + ")"
+    
+    def __repr__(self):
+        return f'Printer("{self.__name}", {self.__speed}, {self.__price})'
+
```
Comment (victoriiaa1) [inline src/printer_lab.py:39]: you can also pick up one tape here

File: src/printer_lab.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+class Printer:
+    manufacturer = "hp"
+    count = 0
+    
+    def __init__(self, name = "OfficeJet Pro 8567e", speed = 35, price = 9500):
```
Comment (victoriiaa1) [inline src/printer_lab.py:5]: remove the spaces before (=) and after

File: src/printer_lab.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+print(repr(pr1))
+
+del pr1
+print(Printer.count)
+del pr2
+print(Printer.count)
+
+print(pr3.get_name(), pr3.get_speed(), pr3.get_price(), sep=" ")
+
```
Comment (victoriiaa1) [inline src/printer_lab.py:65]: here also take 1 ribbon

**Review by victoriiaa1 — COMMENTED** [review]
everything is fine, just fix some bugs on pep 8

File: src/printer_lab.py (hunk 1 above)
Comment (IrynaLutsan) [reply src/printer_lab.py:32]: What do you mean? Remove empty tapes or add them?

**Pull request merged**